python/amico/core.py

```diff
@@ -292,7 +308,23 @@ def fit( self ) :
                         y_est = np.dot( A, x )
                         den = np.sum(y**2)
                         NRMSE[ix,iy,iz] = np.sqrt( np.sum((y-y_est)**2) / den ) if den > 1e-16 else 0
-
+                                            
+                    if self.get_config('doSaveCorrectedDWI') :
```

Comment:
Can we generalize this "correction" also to other models? Does it make sense?
This part seems to be related only to the WaterFree model and I think it breaks with other models (e.g. self.model.d_isos might not be in the model). If so, perhaps we should find a way to save "extra parameters/maps" that are specific to each model.